easybuild/easyconfigs/c/CPLEX/CPLEX-22.10-GCCcore-11.2.0-Python-3.9.6.eb

```diff
@@ -0,0 +1,26 @@
+name = 'CPLEX'
+version = '22.10'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.ibm.com/analytics/cplex-optimizer'
+description = """IBM ILOG CPLEX Optimizer's mathematical programming technology enables
+ analytical decision support for improving efficiency,
+ reducing costs, and increasing profitability."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+# the Academic Initiative version (as used in this file) can be downloaded as described on
+# https://www.ibm.com/developerworks/community/blogs/jfp/entry/cplex_studio_in_ibm_academic_initiative?lang=en
+# a restricted "Community edition" version can be found on
+# https://www-01.ibm.com/software/websphere/products/optimization/cplex-studio-community-edition/
```

Comment:
since we added support for `download_instructions`, it would be good to make use of that here. it accepts a string, and makes it more obvious why an easyconfig failed to build if the sources are not found (since they're not auto-downloaded)